src/pages/profile/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NextPage } from "next";
+import Page from "src/components/Page";
+import ProfileContent from "src/pageContent/Profile";
+
+const Profile: NextPage = () => {
+  return (
+    <Page
```

Comment:
instead of using this, you can try to not render the sub banner like aksara and doesn't need the tailwind config because tbh its pretty bad if you want to assign 1 path to each picture

Reply:
Ini mksd lu yg pass the props to Page component is it?

Reply:
iya, pass url nya instead of tailwind nya nambah 1 config, soalnya entar lama2 jadi kotor tailwindnya